src/fontra/views/editor/panel-user-settings.js

```diff
@@ -68,7 +69,22 @@ export default class UserSettingsPanel extends Panel {
     });
 
     items.push({
-      displayName: "Experimental features",
+      displayName: translate("sidebar.user-settings.language"),
+      controller: languageController,
+      descriptions: [
+        {
+          key: "language",
+          ui: "radio",
+          options: [
+            { key: "en", displayName: translate("language.en") },
```

Comment:
I would love for the language to also always be available in english, maybe even the section header as well. Use case: someone (like me) playing around with languages, and selecting one they don't master, and then can't find their way back :) Maybe add the enlish name in parenthesis after the chinese name? Simplest would be to add it to the translation strings, but perhaps it is better to enforce this in code. What do you think?